### ISSUE TITLE
[MJAR-163] maven jar plugin fails if bundle plugin defined in type 'pom'

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-jar-plugin#314]
**[Eric Charles](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=echarles)** opened **[MJAR-163](https://issues.apache.org/jira/browse/MJAR-163?redirect=false)** and commented

I have the same issue as the one posted here:
http://markmail.org/message/mjsn2xozkcesethl

Running 'mvn install' on the attached pom give:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-jar-plugin:2.4:jar (jar) on project test: Error assembling JAR: Manifest file: /Users/eric/wrk/spc/aos/test/target/osgi/MANIFEST.MF does not exist. -> [Help 1]

Yes, no class to process in a pom, but I would expect maven-jar-plugin to play nice (ignore the abscence of osgi/... file) in such a case.

---

**Affects:** 2.4

**Attachments:**
- [pom.xml](https://issues.apache.org/jira/secure/attachment/12716803/pom.xml) (_2.35 kB_)

2 votes, 4 watchers

**Comment by jira-importer** [comment]
**[Marco Peters](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=marcop)** commented

I have a similar problem. Till now I've used 2.5 but wanted to switch to 3.0.2.
In the previous version I've used useDefaultManifestFile in the configuration. I was able to add this into the plugin configuration of the parent pom and all child modules used this configuration.
Now I have to use manifestFile but this fails in the parent pom. So I would have to declare this configuration in each module. Which is pretty redundant. So I have to stick to 2.5.
